### PR TITLE
Restore profile header and downloads tab

### LINKED ISSUE #1039: Restore profile header
It refers to this 
image

### PR DESCRIPTION
**Pull request by claudiaHash**
Fixes #1039 and #1044 .

Changes proposed in this PR:

- restore `Sales` number in profile header stats
- restore `Downloads` tab

### CODE REVIEW

File: app.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   // return appConfig.metadataCacheUri
   metadataCacheUri:
     process.env.NEXT_PUBLIC_METADATACACHE_URI ||
-    'https://v4.aquarius.oceanprotocol.com',
+    'https://aquariusv4.oceanprotocol.com',
```
Comment (kremalicious) [inline app.config.js:10]: why are we changing this URL again? `v4.aquarius.oceanprotocol.com` is the final URL and it's up, see #1004

File: src/@context/Profile.tsx [hunk 2]
```diff
@@ -186,7 +186,7 @@ function ProfileProvider({
           cancelTokenSource.token
         )
         setAssets(result.results)
-        setAssetsTotal(result.totalResults)
+        setAssetsTotal(result.results.length)
         LoggerInstance.log(
           `[profile] Fetched ${result.totalResults} assets.`,
```
Comment (kremalicious) [inline src/@context/Profile.tsx:191]: this needs to be changed too, based on above change, `result.totalResults` won't work here

File: src/@context/Profile.tsx [hunk 3]
```diff
@@ -224,12 +224,8 @@ function ProfileProvider({
       const tokenOrders = await getUserTokenOrders(accountId, chainIds)
 
       for (let i = 0; i < tokenOrders?.length; i++) {
-        const did = web3.utils
-          .toChecksumAddress(tokenOrders[i].datatoken.address)
-          .replace('0x', 'did:op:')
-        didList.push(did)
+        didList.push(tokenOrders[i].datatoken.address)
```
Comment (kremalicious) [inline src/@context/Profile.tsx:227]: as far as I see, this is not a DID list anymore but a list of Datatoken addresses, so the variable should be named differently

File: src/@utils/subgraph.ts [hunk 4]
```diff
@@ -157,12 +164,10 @@ const UserTokenOrders = gql`
 
 // TODO: counting orders might be enough here to get sales for a user
```
Comment (kremalicious) [inline src/@utils/subgraph.ts:165]: if it's working, remove the TODO

**Comment by kremalicious** [comment]
Checked out some profiles and all numbers check out, great!

One final small thing: when there are no sales the number goes from a displayed `0` to just empty space. Should just stay as `0` in that case, you can see that on your own profile:

Screen Shot 2022-02-21 at 14 27 29

**Comment by claudiaHash** [comment]
I'll investigate this a little, because in my profile I can see this: 

Screenshot from 2022-02-21 16-43-34

**Comment by claudiaHash** [comment]
I'll investigate this a little because I can see the sales number on my profile, which seems to be a correct one [according to the subgraph ](https://v4.subgraph.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph/graphql?query=%7B%0A%20%20user(subgraphError%3A%20deny%20id%3A%220x134b4edd7de8ea003e721e7670800ba97d9af1fd%22)%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20totalSales%0A%20%20%20%20%7D%0A%7D%0A)
Screenshot from 2022-02-21 16-43-34

**Comment by kremalicious** [comment]
very interesting!

So in Firefox this is my setup, without wallet connected or with wallet connected doesn't make difference for me. But activate Polygon Mumbai and you will see the _Sales_ number disappear:

Screen Shot 2022-02-21 at 14 53 38

So I think 2 things:

- double check why some Polygon Mumbai call does not succeed
- make sure calculations and numbers stay even if one subgraph errors out

**Comment by claudiaHash** [comment]
Think i found the problem. You're right, it came out when I used Polygon Mumbai. Just checked with Ropsten after I made the fix and seems to be fine now for my profile and other users's profiles

**Pull request merged**